enterprise/server/backends/userdb/userdb.go

```diff
@@ -518,7 +538,8 @@ func (d *UserDB) createUser(ctx context.Context, tx *db.DB, u *tables.User) erro
 	}
 	emailDomain := emailParts[1]
 
-	if d.env.GetConfigurator().GetAppAddUserToDomainGroup() {
+	// If the user was created via a group - don't add it to a group based on domain.
```

Comment:
I'm having trouble parsing this comment -- can you explain the "created via a group" part?

Reply:
Added some more detail